### DISCUSSION TITLE
Edit the car sprite

### DISCUSSION THREAD

**Discussion by Tetsu13** [djyt/cannonball#143] [Ideas]
is it possible to do it? i'm a ferrari fan and would want to include any other cars, like outrun 2, i own a thrustmaster t80 488 gtb and of course i naturally want to make a 488/458 sprite, where should i start?

**Comment by Beastman1976** [comment]
It would be cool if the original sprite could be updated. The main thing that bugs me is the wheels don't look like they are turning. 

Also be nice if it looked more like an actual testarossa, with the black bars over the lights etc.
